src/Components/Footer/Footer.jsx

```diff
@@ -1,13 +1,21 @@
 import './Footer.css';
 import { Popup } from 'semantic-ui-react';
+import useModal from 'Components/Footer/useModal';
```

Comment:
It beginf with "use" -> Is this a custom hook ?

Reply:
**just checked** Nice !
Tomorrow I'll ask someone random in the group to explain what it does, just to be sure you didn't write misunderstood code in your project :p